lusSTR/snps.py

```diff
@@ -74,13 +73,16 @@ def uas_format(infile, snp_type_arg):
             forward_strand_allele = complement_base(uas_allele)
         else:
             forward_strand_allele = uas_allele
-        if forward_strand_allele in metadata['Alleles']:
+        if data_filt.loc[j, 'Typed Allele?'] == 'No':
+            flag = 'Contains untyped allele'
```

Comment:
Does the `Typed Allele?` column refer to whether there were any reads for that allele?

In any case, space and punctuation in column names can be problematic. If you have just added the column in this PR, I'd recommend using `IsTyped` instead, and boolean values (`True`/`False`) rather than `"Yes"`/`"No"` strings.

Reply:
Or `AlleleIsTyped` or something.

Reply:
The ```Typed Allele?``` column is from the Sample Details Report... it doesn't necessarily indicate an allele with 0 reads, but an allele with reads below the various thresholds, so can have a low number of reads as well as 0 (i.e. is the allele considered to be a real allele). The ```Yes/No``` is read directly from the Sample Details Report, so I'd prefer to leave that as is, but I can change the column name.

Reply:
> The `Typed Allele?` column is from the Sample Details Report

I see. Maybe worth just leaving it in then...